merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
As I said, I am not a big fan of changing the default behavior, because I like it strict. But, that's just a personal preference. The only compromise I see here would be an optional argument for the `delete` operator which tells the delete operator to skip non-existing entries. You know, something like: `(( delete if-present "something" ))`